pkgs/top-level/python-packages.nix

```diff
@@ -261,6 +261,42 @@ in modules // {
     sourceRoot = "letsencrypt-${version}/acme";
   };
 
+  acme-tiny = buildPythonPackage rec {
+    name = "acme-tiny-${rev}";
```

Comment:
Please change version to `date`. See https://nixos.org/nixpkgs/manual/#sec-package-naming